src/MeetingsSDKAdapter.js

```diff
@@ -79,24 +80,23 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       sendShare: false,
     };
 
-    sdkMeeting
-      .getMediaStreams(mediaSettings)
-      .then(([localStream, localShare]) => {
-        sdkMeeting
-          .addMedia({
-            localShare,
-            localStream,
-            mediaSettings,
-          })
-          .catch((error) => {
-            // eslint-disable-next-line no-console
-            console.error(`Unable to add local media to meeting "${ID}"`, error);
-          });
-      })
-      .catch((error) => {
+    try {
+      const [localStream, localShare] = await sdkMeeting.getMediaStreams(mediaSettings);
+
+      try {
+        sdkMeeting.addMedia({
```

Comment:
I don't think you will catch a promise thrown error in a try/catch block without adding `async sdkMeeting.addMedia`.